reference/halyard/component-sizing.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: single
+title:  "Component Sizing"
+sidebar:
+  nav: reference
+redirect_from: /docs/component-sizing
```

Comment:
No need to include a `redirect_from`. This was only needed for old docs that transferred from the readme.io site.